### PR TITLE
Added missing headTitle to test-page

### LINKED ISSUE #127: Missing headTitle in test/index.js
... which causes jsperf to fallback to the generic `<title>`

The headTitle should be set to the same as the test-title(and should not mention the revision, if we are stricly trying to mimic jsPerf v1)

### PR DESCRIPTION
**Pull request by madskonradsen**
Closes #127

### CODE REVIEW

**Comment by maxbeatty** [comment]
Is there a test we can add so we're alerted if the title goes missing again?

**Comment by madskonradsen** [comment]
I could make tests for all the pages? I don't think it would be possible to cover all headTitles with just one test...

**Comment by maxbeatty** [comment]
for this PR, only worried about the test page title. you should be able to assert [here](https://github.com/jsperf/jsperf.com/blob/f9f05fbccd178926c9ae7956a284799ef896b508/test/server/web/test/index.js#L129) that ["Oh Yea"](https://github.com/jsperf/jsperf.com/blob/f9f05fbccd178926c9ae7956a284799ef896b508/test/server/web/test/index.js#L112) is in `<title>`

File: server/web/edit/index.js [hunk 1]
```diff
@@ -64,7 +65,11 @@ exports.register = function (server, options, next) {
         }
         let page = Hoek.applyToDefaults(defaults.testPageContext, request.payload, true);
         Hoek.merge(page, errObj);
-        reply.view('edit/index', {page: page, authorized: true}).code(400);
+        reply.view('edit/index', {
+          headTitle: page.title,
```
Comment (maxbeatty) [inline server/web/edit/index.js]: can you add a test for the edit page similar to test page? would be nice if it's verified for both 200 and 400 scenarios

**Comment by maxbeatty** [comment]
looks good. will you please rebase so I can merge?

**Comment by madskonradsen** [comment]
Would this suffice? I'm not totally up-to-date on the lingo :)

**Pull request closed without merging**